README.md

```diff
@@ -191,10 +98,10 @@ kg = KnowledgeGraph(
 kg.process_sources(sources)
 ```
 
-You can update the KG at any time by processing more sources with the `process_sources` method.
+### Optional: Configuring your prompts
+When creating your Knowledge Graph (KG) agent, you can customize the prompts to tailor its behavior.
 
-#### Configurable Prompts
-When creating your Knowledge Graph (KG) agent, you can customize the prompts to tailor its behavior. This step is optional but can enhance functionality.
+💡 This step is optional but can enhance functionality.
 
 There are five types of prompts:

```

Comment:
a lot to config, not sure if you want to mention this, could be intimidating.

Reply:
same as above, I think we can remove it for now, agree with me?

Reply:
moved to the end of the readme